README.md

```diff
@@ -104,70 +104,128 @@ that more automatic in the end.
+                        files
+  --isCorr              Run with SFs
+  --isJERC              JER/JEC implemented to jet
+  --executor {iterative,futures,parsl/slurm,parsl/condor,parsl/condor/naf_lite,dask/condor,dask/slurm,dask/lpc,dask/lxplus,dask/casa}
+                        The type of executor to use (default: futures). 
+  -j WORKERS, --workers WORKERS
+                        Number of workers (cores/threads) to use for multi- worker executors (e.g. futures or condor) (default:
+                        12)
+  -s SCALEOUT, --scaleout SCALEOUT
+                        Number of nodes to scale out to if using slurm/condor.
+                        Total number of concurrent threads is ``workers x
+                        scaleout`` (default: 6)
+  --memory MEMORY       Memory used in jobs  ``(default: 4GB)
+  --disk DISK           Disk used in jobs  ``(default: 4GB)
+  --voms VOMS           Path to voms proxy, made accessible to worker nodes.
+                        By default a copy will be made to $HOME.
+  --chunk N             Number of events per process chunk
+  --retries N           Number of retries for coffea processor
+  --validate            Do not process, just check all files are accessible
+  --skipbadfiles        Skip bad files.
+  --only ONLY           Only process specific dataset or file
+  --limit N             Limit to the first N files of each dataset in sample
+                        JSON
+  --max N               Max number of chunks to run in total
+```
+</p>
+</details>
+
+Roadmap for running the tool
+
+1. Is the `.json` files ready? If not create it through [Make the json files](#make-the-json-files) section with naming scheme
+
+2. If you want JERC and SFs implemented, edit the correction config under `BTVNanoCommissioning/src/utils/AK4_parameters.py`, add a new `dict` contains correction files in dedicated campaign.
+
+3. If the JERC file missing `jec_compiled.pkl.gz` in the `data/JME/${campaign}` directory, create it through [Create compiled JERC file](#create-compiled-jerc-filepklgz)
+
+4. Run the workflows with dedicated input and campaign name. example commands for run3 could be found [here](#commands-for-different-phase-space)
+
+5. Once you get `.coffea` file, you can make plots via the [plotting scripts](#plotting-code), if the xsection for your sample is missing, please add to `src/BTVNanoCommissioning/helpers/xsection.py`
+
+### commands for different phase space
+
+After small test, you can run the whole campaign according the phase space.
+
+#### b-SFs 
 
 <details><summary>details</summary>
 <p>
 
 - Dileptonic ttbar phase space : check performance for btag SFs, muon channel
 
 ```
-python runner.py --workflow (e)ttdilep_sf --json metadata/94X_doublemu_PFNano.json
+ python runner.py --workflow ttdilep_sf --json metadata/data_Winter22_emu(mumu)_BTV_Run3_2022_Comm_v1.json  --campaign Winter22Run3 --year 2022  (-j 4 --executor ${scaleout_site} -s 500) 
```

Comment:
remove `mumu` in json file name
do we need to give here an example for `-j` and `-s`? This could be understood as that one needs to use `-j 4 -s 500` as optimal choice, which is not the case depending on the chosen executor?

Reply:
you are right, I think I would remove the scaleout options since this really depends on site. user might need to choose themselves